client/components/time-since/README.md

```diff
@@ -18,3 +18,4 @@ Over a week:
 ## Props
 
 - `date`: Date, The date to show
+- `defaultFormat`: String, Override the formatting of dates older than 1 week.
```

Comment:
As this is more of an override, I'm not convinced we should call it 'default' :) How about we can just call this prop `dateFormat`?

Reply:
I found it tough to name since it's only for the fall-through case but `dateFormat` works perfectly! Thanks @bluefuton!